sdk/keyvault/azure-security-keyvault-certificates/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features Added
 
 ### Breaking Changes
+- Updated CreateCertificateOperation.PollUntilDone method (returned from StartCreateCertificate call)  to return the operation status instead of the newly created certificate.
```

Comment:
```suggestion

- Updated `CreateCertificateOperation.PollUntilDone()`  (returned from `StartCreateCertificate()`)  to return the operation status instead of the newly created certificate.
```

Reply:
Don't care too much about the wording, but there must be one empty line between the title `### Breaking Changes` and the bullet

Reply:
done